src/Npgsql/PgPassEntry.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Npgsql
+{
+    /// <summary>
+    /// Represents a hostname, port, database, username, and password combination that has been retrieved from a .pgpass file
+    /// </summary>
+    class PgPassEntry
+    {
+        #region Fields and Properties
+
+        /// <summary>
+        /// Hostname parsed from the .pgpass file
+        /// </summary>
+        public string Hostname { get; }
```

Comment:
Please mark all the members of this class internal rather than public (both methods and properties)